// RUN_PIPELINE_TILL: FRONTEND

@MetroStation
interface <!METRO_STATION_NOT_ON_CLASS!>MyInterface<!>

@MetroStation
abstract class <!METRO_STATION_NOT_ON_CLASS!>MyAbstractClass<!>

@MetroStation
object <!METRO_STATION_NOT_ON_CLASS!>MyObject<!>

@MetroStation
enum class <!METRO_STATION_NOT_ON_CLASS!>MyEnum<!>

@MetroStation
class MyClass

/* GENERATED_FIR_TAGS: classDeclaration, classReference, enumDeclaration, interfaceDeclaration, objectDeclaration */
